README.md

```diff
@@ -140,13 +140,23 @@ In order to use it, you need to create the blacklist table in a migration:
 def change
   create_table :jwt_blacklist do |t|
     t.string :jti, null: false
+    t.datetime :exp, null: false
   end
   add_index :jwt_blacklist, :jti
 end
 ```
-
 For performance reasons, it is better if the `jti` column is an index.
 
+Note, if you used the blacklist strategy in a previous version you may not have the field *exp.* If not run the following migration:
+```ruby
+class AddExpirationTimeToJWTBlacklist < ActiveRecord::Migration
+  def change
+    add_column :jwt_blacklist, :exp, :datetime, null: false, default: Time.now
```

Comment:
As I told in the other comment, the default value is only important for our test environment. Usually, people will be using something more robust that sqlite and that will support datetimes without default values. I would remove the `default: Time.now` from the recommended migration in the README. Even if it is not that important because the revocation strategy always will populate the `exp` itself, conceptually is awkward to recommend "all tokens to be expired now".